api/notd/api/api_v1.py

```diff
@@ -215,6 +216,13 @@ async def list_account_tokens(accountAddress: str, limit: Optional[int] = None,
         tokenKeys = await notdManager.list_account_tokens(accountAddress=accountAddress, limit=limit, offset=offset)
         return GetAccountTokensResponse(tokens=(await responseBuilder.collection_tokens_from_token_keys(tokenKeys=tokenKeys)))
 
+    @router.get('/accounts/{accountAddress}/tokens/delegate', response_model=GetAccountTokensWithDelegateTokensResponse)
```

Comment:
should be list. also the url isnt great, it should be something like `delegated-tokens` so the function / classes become `ListAccountDelegatedTokens`